libcontainer/container_linux.go

```diff
@@ -1896,16 +1896,17 @@ func (c *linuxContainer) currentState() (*State, error) {
 		startTime           uint64
 		externalDescriptors []string
 		pid                 = -1
+		intelRdtPath        string
```

Comment:
So, it seems that getting the path is trivial, and instead of adding it to various structures we should remove it and rely on what `c.intelRdtManager.GetPath()` returns. This includes the current code, not just this patch.

I would very much like to see a nice clean refactor of intelrdt code first, then adding more stuff.